ote/src/clj/ote/services/external.clj

```diff
@@ -2,13 +2,26 @@
   "CKAN-integration tasks. Functions to invoke CKAN API calls from OTE."
   (:require [com.stuartsierra.component :as component]
             [ote.components.http :as http]
+            [ote.db.transport-service :as t-service]
+            [ote.db.modification :as modification]
             [org.httpkit.client :as httpkit]
             [compojure.core :refer [routes GET POST DELETE]]
             [taoensso.timbre :as log]
             [ote.authorization :as authorization]
             [clojure.string :as str]
             [clojure.data.csv :as csv]
-            [clojure.string :as s]))
+            [clojure.string :as s]
+            [specql.core :as specql]))
+
+(defn valid-csv-header?
+  "Ensure that there is at least 2 elements in header"
+  [header]
+  (>= (count header) 2))
+
+(defn valid-business-id? [value]
+  (let [pattern  #"\d{7}-\d"]
+  (if  (re-matches pattern value)
```

Comment:
Avoid `(if condition true false)` pattern.

Often you can just use `condition` by itself or wrap it with `boolean` to force a `true`/`false` value.

```
(defn valid-business-id? [business-id]
  (boolean (re-matches #"\d{7}-\d" business-id)))
```